__tests__/app.test.js

```diff
@@ -53,3 +53,41 @@ describe("/api endpoints", () => {
       });
   });
 });
+
+describe("/api/articles", () => {
```

Comment:
You are using the endpoint /api/articles as your description however this isn't the endpoint you are testing for. I would change this and be more specific with your descriptions going forward